### PR TITLE
docs:  Chinese doc of architecture

### PR DESCRIPTION
**Pull request by MachaelLee**
* Chinese doc of architecture
* icon of language changing

### CODE REVIEW

File: docs/src/en/architecture.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ The model of CeresDB processing data is `WAL` + `MemTable` that the recent writt
 
 Now two implementations of `WAL` are provided for stand-alone and distributed mode:
 - For stand-alone mode, `WAL` is based on `RocksDB` and data is persisted on the local disk.
-- For distributed mode, `WAL` is required as a distributed component and to be responsible for reliability of the newly written data, so now we provide an implementation based on [OceanBase](https://github.com/oceanbase/oceanbase) and in our roadmap a more lightweight implementation will be provided.
+- For distributed mode, `WAL` is required as a distributed component and to be responsible for reliability of the newly written data, so now we provide an implementation based on [OceanBase](https://github.com/oceanbase/oceanbase).
```
Comment (ShiKaiWi) [inline docs/src/en/architecture.md:114]:
```suggestion
- For distributed mode, `WAL` is required as a distributed component and to be responsible for durability of the newly written data, so now we provide an implementation based on [OceanBase](https://github.com/oceanbase/oceanbase).
```

File: docs/src/cn/architecture.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
-# Introduction to CeresDB's Architecture
-## Target
-- Provide the overview of CeresDB to the developers who want to know more about CeresDB but have no idea where to start.
-- Make a brief introduction to the important modules of CeresDB and the connections between these modules but details about their implementations are not be involved.
+# CeresDB 架构介绍
 
-## Motivation
-CeresDB is a timeseries database. However, CeresDB's goal is to handle both timeseries and analytic workloads compared with the traditional ones, which usually have a poor performance in handling analytic workloads.
+## 目标
```
Comment (ShiKaiWi) [inline docs/src/cn/architecture.md:3]:
```suggestion
## 本文目标
```

File: docs/src/cn/architecture.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 
-In the traditional timeseries database, the `Tag` columns (InfluxDB calls them `Tag` and Prometheus calls them `Label`) are normally indexed by generating an inverted index. However, it is found that the cardinality of `Tag` varies in different scenarios. And in some scenarios the cardinality of `Tag` is very high, and it takes a very high cost to store and retrieve the inverted index. On the other hand, it is observed that scanning+pruning often used by the analytical databases can do a good job to handle such these scenarios.
+- 为想了解更多关于 CeresDB 但不知道从何入手的开发者提供 CeresDB 的概览
+- 简要介绍 CeresDB 的主要模块以及这些模块之间的联系，但不涉及它们实现的细节
 
-The basic design idea of CeresDB is to adopt a hybrid storage format and the corresponding query method for a better performance in processing both timeseries and analytic workloads.
+## 动机
+
+CeresDB 是一个时间序列数据库。但是，与传统数据库相比，CeresDB 的目标是能够同时处理时间序列和分析工作负载，并提供较高性能。
```
Comment (ShiKaiWi) [inline docs/src/cn/architecture.md:10]:
```suggestion
CeresDB 是一个时间序列数据库。但是，与传统数据库相比，CeresDB 的目标是能够同时处理时序型和分析型两种模式的数据，并提供较高性能。
```

File: docs/src/cn/architecture.md [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+
+在传统的时间序列数据库中，`Tag` 列（ `InfluxDB` 称之为 `Tag`，`Prometheus` 称之为 `Label`）通常通过生成倒排索引进行索引。但是，人们发现 `Tag` 的基数在不同的场景中是不一样的。在某些场景中，`Tag` 的基数非常高，存储和检索倒排索引要付出很高的代价。另一方面，分析数据库常用的扫描+修剪方法可以很好地处理这些情况。
```
Comment (ShiKaiWi) [inline docs/src/cn/architecture.md:12]:
```suggestion
在经典的时间序列数据库中，`Tag` 列（ `InfluxDB` 称之为 `Tag`，`Prometheus` 称之为 `Label`）通常会对其生成倒排索引。但是，在实际场景中，`Tag` 的基数在不同的场景中是不一样的。在某些场景中，`Tag` 的基数非常高，存储和检索倒排索引要为此付出很高的代价。而另一方面，分析型数据库常用的扫描 + 剪枝方法，可以比较高效地处理这样的情况。
```

And update the `traditional` in the English version to `classic`.

File: docs/src/cn/architecture.md [hunk 5]
```diff
@@ -1,16 +1,20 @@
...
+
+CeresDB 的基本设计理念是采用混合存储格式和相应的查询方法，以便更好地处理时间序列和分析工作负载。
```
Comment (ShiKaiWi) [inline docs/src/cn/architecture.md:14]:
```suggestion
CeresDB 的基本设计理念是采用混合存储格式和相应的查询方法，去处理时序型数据和分析型数据，获得更好的性能。
```

File: docs/src/cn/architecture.md [hunk 6]
```diff
@@ -54,148 +54,144 @@ The basic design idea of CeresDB is to adopt a hybrid storage format and the cor
...
+### SQL 层
 
-module path: https://github.com/CeresDB/ceresdb/tree/main/sql
+模块路径：https://github.com/CeresDB/ceresdb/tree/main/sql
 
-SQL layer takes responsibilities for parsing sql and generating the plan.
+SQL 层负责解析 SQL 并生成查询计划。
 
-Based on [sqlparser](https://github.com/sqlparser-rs/sqlparser-rs) a sql dialect, which introduces some key concepts including `Tag` and `Timestamp`, is provided for processing timeseries data. And by utilizing [DataFusion](https://github.com/apache/arrow-datafusion) the planner can generate not only normal logical plans but also custom ones, such as plans for `PromQL`.
+CeresDB 基于 [sqlparser](https://github.com/sqlparser-rs/sqlparser-rs) 提供了一种 sql 方言，为了更好的适配时序数据，引入一些概念，包括 Tag 和 Timestamp。此外，利用 [DataFusion](https://github.com/apache/arrow-datafusion)，CeresDB 不仅可以生成常规的逻辑计划，还可以生成自定义的计划来实现时序场景要求的特殊算子，例如为了适配 PromQL 协议而做的工作就是利用了这个特性。
```
Comment (MachaelLee) [inline docs/src/cn/architecture.md:73]:
```suggestion
CeresDB 基于 [sqlparser](https://github.com/sqlparser-rs/sqlparser-rs) 提供了一种 SQL 方言，为了更好的适配时序数据，引入一些概念，包括 Tag 和 Timestamp。此外，利用 [DataFusion](https://github.com/apache/arrow-datafusion)，CeresDB 不仅可以生成常规的逻辑计划，还可以生成自定义的计划来实现时序场景要求的特殊算子，例如为了适配 PromQL 协议而做的工作就是利用了这个特性。
```

File: docs/src/cn/architecture.md [hunk 7]
```diff
@@ -54,148 +54,144 @@ The basic design idea of CeresDB is to adopt a hybrid storage format and the cor
...
-Based on [sqlparser](https://github.com/sqlparser-rs/sqlparser-rs) a sql dialect, which introduces some key concepts including `Tag` and `Timestamp`, is provided for processing timeseries data. And by utilizing [DataFusion](https://github.com/apache/arrow-datafusion) the planner can generate not only normal logical plans but also custom ones, such as plans for `PromQL`.
+CeresDB 基于 [sqlparser](https://github.com/sqlparser-rs/sqlparser-rs) 提供了一种 SQL 方言，为了更好的适配时序数据，引入一些概念，包括 Tag 和 Timestamp。此外，利用 [DataFusion](https://github.com/apache/arrow-datafusion)，CeresDB 不仅可以生成常规的逻辑计划，还可以生成自定义的计划来实现时序场景要求的特殊算子，例如为了适配 PromQL 协议而做的工作就是利用了这个特性。
 
 ### Interpreter
 
-module path: https://github.com/CeresDB/ceresdb/tree/main/interpreters
+模块路径：https://github.com/CeresDB/ceresdb/tree/main/interpreters
 
-The `Interpreter` module encapsulates the SQL `CRUD` operations. Actually, a sql received by CeresDB will be parsed, converted into the query plan and then executed in some specific interpreter, such as `SelectInterpreter`, `InsertInterpreter` and etc.
+`Interpreter` 模块封装了 SQL 的 `CRUD` 操作。在查询流程中，一个 sql 语句会经过解析，生成出对应的查询计划，然后便会在特定的解释器中执行，例如 `SelectInterpreter`、`InsertInterpreter` 等。
```
Comment (MachaelLee) [inline docs/src/cn/architecture.md:79]:
```suggestion
`Interpreter` 模块封装了 SQL 的 `CRUD` 操作。在查询流程中，一个 SQL 语句会经过解析，生成出对应的查询计划，然后便会在特定的解释器中执行，例如 `SelectInterpreter`、`InsertInterpreter` 等。
```

File: docs/src/cn/architecture.md [hunk 8]
```diff
@@ -54,148 +54,144 @@ The basic design idea of CeresDB is to adopt a hybrid storage format and the cor
...
 #### MemTable
 
-module path: https://github.com/CeresDB/ceresdb/tree/main/analytic_engine/src/memtable
+模块路径：https://github.com/CeresDB/ceresdb/tree/main/analytic_engine/src/memtable
 
-`Memtable` is used to store the newly written data and after a certain amount of data is accumulated, CeresDB organizes the data in `MemTable` into a query-friendly storage format (`SST`) and stores it to the persistent device. `MemTable` is readable before it gets persisted (flushed).
+由于 `WAL` 无法提供高效的查询，因此新写入的数据会存储一份到 `Memtable` 用于查询，并且在积累了一定数量后，CeresDB 将 `MemTable` 中的数据组织成便于查询的存储格式（`SST`）并存储到持久化设备中。
 
-The current implementation of `MemTable` is based on [agatedb's skiplist](https://github.com/tikv/agatedb/blob/8510bff2bfde5b766c3f83cf81c00141967d48a4/skiplist). It allows concurrent reads and writes and can control memory usage based on [Arena](https://github.com/CeresDB/ceresdb/tree/main/components/skiplist).
+MemTable 的当前实现基于 [agatedb 的 skiplist](https://github.com/tikv/agatedb/blob/8510bff2bfde5b766c3f83cf81c00141967d48a4/skiplist)。它允许并发读取和写入，并且可以根据 [Arena](https://github.com/CeresDB/ceresdb/tree/main/components/skiplist) 控制内存使用。
```
Comment (MachaelLee) [inline docs/src/cn/architecture.md:136]:
```suggestion
MemTable 的当前实现基于 [AgateDB 的 skiplist](https://github.com/tikv/agatedb/blob/8510bff2bfde5b766c3f83cf81c00141967d48a4/skiplist)。它允许并发读取和写入，并且可以根据 [Arena](https://github.com/CeresDB/ceresdb/tree/main/components/skiplist) 控制内存使用。
```

File: docs/src/cn/architecture.md [hunk 9]
```diff
@@ -54,148 +54,144 @@ The basic design idea of CeresDB is to adopt a hybrid storage format and the cor
...
 
-- Table schema and table options;
-- The sequence number where the newest flush finishes;
-- The information of `SST`, such as `SST` path.
+- 表的结构和表的参数选项；
+- 最新 `Flush` 过的 sequence number；
+- 表的所有 `SST` 文件的信息。
 
-Now the `Manifest` is based on `WAL` (this is a different instance from the `WAL` mentioned above for newly written data) and in order to avoid infinite expansion of metadata (actually every `Flush` leads to an update on sst information), `Snapshot` is also introduced to clean up the history of metadata updates.
+现在 `Manifest` 是基于 `WAL` 和 `Object Store` 来实现的，新的改动会直接写入到 `WAL`，而为了避免元数据无限增长（实际上每次 `Flush` 操作都会触发更新），会对其写入的记录做快照，生成的快照会被持久化道 `Object Store`。
```
Comment (MachaelLee) [inline docs/src/cn/architecture.md:162]:
```suggestion
现在 `Manifest` 是基于 `WAL` 和 `Object Store` 来实现的，新的改动会直接写入到 `WAL`，而为了避免元数据无限增长（实际上每次 `Flush` 操作都会触发更新），会对其写入的记录做快照，生成的快照会被持久化到 `Object Store`。
```

File: docs/src/en/architecture.md [hunk 10]
```diff
@@ -121,18 +121,17 @@ module path: https://github.com/CeresDB/ceresdb/tree/main/wal
...
 
 #### MemTable
 
 module path: https://github.com/CeresDB/ceresdb/tree/main/analytic_engine/src/memtable
 
-`Memtable` is used to store the newly written data and after a certain amount of data is accumulated, CeresDB organizes the data in `MemTable` into a query-friendly storage format (`SST`) and stores it to the persistent device. `MemTable` is readable before it gets persisted (flushed).
+For `WAL` can't provide efficient data retrieval, the newly written data is also stored in `Memtable` for efficient data retrieval, after a certain amount of data is reached, CeresDB organizes the data in `MemTable` into a query-friendly storage format (`SST`) and stores it to the persistent device.
 
 The current implementation of `MemTable` is based on [agatedb's skiplist](https://github.com/tikv/agatedb/blob/8510bff2bfde5b766c3f83cf81c00141967d48a4/skiplist). It allows concurrent reads and writes and can control memory usage based on [Arena](https://github.com/CeresDB/ceresdb/tree/main/components/skiplist).
```
Comment (MachaelLee) [inline docs/src/en/architecture.md:136]:
```suggestion
The current implementation of `MemTable` is based on [AgateDB's skiplist](https://github.com/tikv/agatedb/blob/8510bff2bfde5b766c3f83cf81c00141967d48a4/skiplist). It allows concurrent reads and writes and can control memory usage based on [Arena](https://github.com/CeresDB/ceresdb/tree/main/components/skiplist).
```

**Review by ShiKaiWi — APPROVED** [review]
LGTM

**Pull request merged**